templates/docker-compose.template

```diff
@@ -48,20 +43,17 @@ services:
       - {{I2C_DEVICE}}:{{I2C_DEVICE}}
     restart: on-failure
     environment:
-      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
       - I2C_DEVICE={{I2C_DEVICE}}
 
   diagnostics:
     image: nebraltd/hm-diag:{{DIAGNOSTICS_VERSION}}
     depends_on:
-      - helium-miner
-      - dbus-session
+      - gateway-rs
     environment:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
       - FIRMWARE_SHORT_HASH={{ENV.FIRMWARE_SHORT_HASH}}
       - DIAGNOSTICS_VERSION={{DIAGNOSTICS_VERSION}}
-      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
```

Comment:
This is still required

Reply:
I wasn't sure about that. @pritamghanghas What do you think?

Reply:
system bus is still needed.

Reply:
I think we need it for Bluetooth / LTE / WiFi